Api changes

memargonz opened the pull request:
modified POST and GET to process Parties involved in an accident

Commit:
added visual features and interface of party details

Commit:
added visual features and interface of party details

Commit:
Party dialog initializations

Commit:
Party dialog initializations

Commit:
Merge branch 'visual-features' of ssh://github.com/ianrbaguio/crash-app into visual-features

Rebased from main then merged into visual-feature branch

Commit:
Api changes- POST and Get with parties

Commit:
Api changes- added gitignore entries

ianrbaguio reviewed (approved):
👍

ianrbaguio commented on crash/src/app/accident-report-container/intake/intake.component.ts:
```diff
@@ -158,6 +206,7 @@ get dynamicParties() {
 }
 
 onSubmit(): void{
+  console.log(this.PartyFields.length)
```
Please remove console logs

ianrbaguio commented on crash/src/app/accident-report-container/partydialog/partydialog.component.ts:
```diff
@@ -0,0 +1,102 @@
+}
+onSubmit() {
+// code to save the party dialog
+ console.log(this.form.get('PartyLastName')!.value)
```
Please remove console log

ianrbaguio commented on crash/src/app/accident-report-container/partydialog/partydialog.component.ts:
```diff
@@ -0,0 +1,102 @@
+}
+
+
+console.log(this.result.PartyFirstName);
```
Please remove console log

ianrbaguio commented on crash/src/app/accident-report-container/partydialog/partydialog.component.ts:
```diff
@@ -0,0 +1,102 @@
+        private cdf: ChangeDetectorRef ) 
+{
+ 
+    console.log("input data " +data)
```
Please remove console log

ianrbaguio commented on crash/src/app/view-accidents-container/view-map-accidents/search-map.component.ts:
```diff
@@ -265,6 +306,14 @@ export class SearchMapComponent  implements OnInit {
+      this.rectangle.getBounds()?.toJSON().west,
+     );
+
+       console.log( this.rectangle.getBounds()?.toJSON)
```
Please remove console log

ianrbaguio reviewed (changes requested):
Please remove console logs

Commit:
resolve reviewed issues by Ian

memargonz committed:
api changes aug 30

memargonz committed:
Merge branch 'main' into api-changes

memargonz committed:
modified:   api/crash/.gitignore
	new file:   api/crash/.vs/ProjectEvaluation/crash.metadata.v7.bin
	new file:   api/crash/.vs/ProjectEvaluation/crash.projects.v7.bin
	new file:   api/crash/.vs/crash/DesignTimeBuild/.dtbcache.v2
	new file:   api/crash/.vs/crash/FileContentIndex/036e040d-f602-4e3b-9c93-158df5a27b2f.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/17b31021-babd-45ef-9d38-243d22fed296.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/19b7b2e9-d903-408a-a4c4-86545ad876cc.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/30370e26-9149-4cfd-820e-0f554a25ca8e.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/75fd2d2d-9e2d-4b12-9ed2-f18bcc4a115a.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/7a38c890-9799-476a-a0ab-bd25f0ebd0bf.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/b5b391ba-ceb9-46ea-b1f5-d39bfb99d3df.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/bb7bd394-ca93-47c0-88a3-fe2f24808e01.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/d60caeb6-9782-48b0-a8c8-f0297e8600d8.vsidx
	new file:   api/crash/.vs/crash/FileContentIndex/f19983d6-8258-450d-ba31-4a0a16850920.vsidx
	modified:   api/crash/.vs/crash/config/applicationhost.config
	new file:   api/crash/.vs/crash/v17/.futdcache.v2
	modified:   api/crash/.vs/crash/v17/.suo
	new file:   api/crash/.vs/crash/v17/DocumentLayout.json
	new file:   api/crash/Commands/CreateAccident.cs
	new file:   api/crash/Commands/UploadImages.cs
	modified:   api/crash/Controllers/AccidentsController.cs
	modified:   api/crash/Crash.csproj
	modified:   api/crash/Database/AppDbContext.cs
	new file:   api/crash/Handlers/Command/CreateAccidentHandler.cs
	new file:   api/crash/Handlers/Command/UploadImagesHandler.cs
	modified:   api/crash/Handlers/Query/GetAccidentsQueryHandler.cs
	new file:   api/crash/Handlers/Query/GetAccidentsQueryHandler.cs.bak
	new file:   api/crash/Handlers/Query/GetImagesQueryHandler.cs
	new file:   api/crash/Misc/MappingProfile.cs
	new file:   api/crash/Misc/RequestResponseLoggingMiddleware.cs
	modified:   api/crash/Models/Dtos/AccidentDto.cs
	new file:   api/crash/Models/Dtos/AccidentImageDto.cs
	new file:   api/crash/Models/Dtos/ImageDto.cs
	new file:   api/crash/Models/Dtos/PartyDto.cs
	modified:   api/crash/Models/Entities/Accident.cs
	new file:   api/crash/Models/Entities/Image.cs
	new file:   api/crash/Models/Entities/Party.cs
	modified:   api/crash/Program.cs
	new file:   api/crash/Queries/GetAccidentByIdQuery.cs
	new file:   api/crash/Queries/GetAccidentsByRegionQuery.cs
	new file:   api/crash/Queries/GetImagesByAccidentIdQuery.cs
	new file:   api/crash/Repositories/AccidentRepository.cs
	new file:   api/crash/Repositories/IRepositories/IAccidentRepository.cs
	modified:   api/crash/appsettings.Development.json
	deleted:    api/crash/appsettings.json
	new file:   api/crash/sql.txt
	modified:   crash/.data/init.js
	modified:   crash/README.md
	modified:   crash/package-lock.json
	modified:   crash/package.json
	new file:   crash/src/app/accident-details-container/accident-details-container.component.html
	new file:   crash/src/app/accident-details-container/accident-details-container.component.scss
	new file:   crash/src/app/accident-details-container/accident-details-container.component.spec.ts
	new file:   crash/src/app/accident-details-container/accident-details-container.component.ts
	new file:   crash/src/app/accident-details-container/accident-details/accident-details.component.html
	new file:   crash/src/app/accident-details-container/accident-details/accident-details.component.scss
	new file:   crash/src/app/accident-details-container/accident-details/accident-details.component.spec.ts
	new file:   crash/src/app/accident-details-container/accident-details/accident-details.component.ts
	new file:   crash/src/app/accident-details-container/accident-mapdetails/accidentmap.component.html
	new file:   crash/src/app/accident-details-container/accident-mapdetails/accidentmap.component.scss
	new file:   crash/src/app/accident-details-container/accident-mapdetails/accidentmap.component.ts
	new file:   crash/src/app/accident-details-container/accident-mapdetails/map.component.ts.bak
	modified:   crash/src/app/accident-report-container/intake/intake.component.html
	new file:   crash/src/app/accident-report-container/intake/intake.component.html.bak
	modified:   crash/src/app/accident-report-container/intake/intake.component.ts
	new file:   crash/src/app/accident-report-container/intake/intake.component.ts.bak
	modified:   crash/src/app/accident-report-container/map/map.component.ts
	new file:   crash/src/app/accident-report-container/map/map.component.ts.bak
	new file:   crash/src/app/accident-report-container/partydialog/partydialog.component.html
	new file:   crash/src/app/accident-report-container/partydialog/partydialog.component.scss
	new file:   crash/src/app/accident-report-container/partydialog/partydialog.component.spec.ts
	new file:   crash/src/app/accident-report-container/partydialog/partydialog.component.ts
	modified:   crash/src/app/accident-report-container/upload-images/upload.component.html
	new file:   crash/src/app/accident-report-container/upload-images/upload.component.html.bak
	modified:   crash/src/app/accident-report-container/upload-images/upload.component.ts
	new file:   crash/src/app/accident-report-container/upload-images/upload.component.ts.bak
	modified:   crash/src/app/app.routes.ts
	modified:   crash/src/app/crash.service.ts
	new file:   crash/src/app/crash.service.ts.bak
	new file:   crash/src/app/data/accidents.json
	modified:   crash/src/app/home-container/home/home.component.html
	modified:   crash/src/app/home-container/home/home.component.scss
	modified:   crash/src/app/home-container/home/home.component.ts
	new file:   crash/src/app/view-accidents-container/view-accidents-container.component.html
	new file:   crash/src/app/view-accidents-container/view-accidents-container.component.scss
	new file:   crash/src/app/view-accidents-container/view-accidents-container.component.spec.ts
	new file:   crash/src/app/view-accidents-container/view-accidents-container.component.ts
	new file:   crash/src/app/view-accidents-container/view-accidents/view-accidents.component.html
	new file:   crash/src/app/view-accidents-container/view-accidents/view-accidents.component.scss
	new file:   crash/src/app/view-accidents-container/view-accidents/view-accidents.component.spec.ts
	new file:   crash/src/app/view-accidents-container/view-accidents/view-accidents.component.ts
	new file:   crash/src/app/view-accidents-container/view-map-accidents/search-map.component.html
	new file:   crash/src/app/view-accidents-container/view-map-accidents/search-map.component.scss
	new file:   crash/src/app/view-accidents-container/view-map-accidents/search-map.component.ts
	new file:   crash/src/app/view-accidents-container/view-map-detail/mapdialog.component.html
	new file:   crash/src/app/view-accidents-container/view-map-detail/mapdialog.component.ts
	new file:   crash/src/app/view-accidents-container/view-map-detail/mpdialog.component.scss
	new file:   crash/src/assets/images/crash.jpg
	new file:   crash/src/assets/images/crash.png
	modified:   crash/src/index.html
	modified:   crash/src/shared/header/header.component.html
	modified:   crash/tsconfig.json
	new file:   node_modules/.package-lock.json
	new file:   node_modules/guid-typescript/README.md
	new file:   node_modules/guid-typescript/dist/guid.d.ts
	new file:   node_modules/guid-typescript/dist/guid.js
	new file:   node_modules/guid-typescript/package.json
	new file:   package-lock.json
	new file:   package.json

memargonz committed:
Merge branch 'api-changes' of ssh://github.com/ianrbaguio/crash-app into api-changes

	modified:   api/crash/.vs/crash/v17/DocumentLayout.json
	modified:   crash/src/app/accident-report-container/intake/intake.component.html
	modified:   crash/src/app/accident-report-container/intake/intake.component.ts
	modified:   crash/src/app/accident-report-container/partydialog/partydialog.component.html
	modified:   crash/src/app/accident-report-container/partydialog/partydialog.component.ts
	modified:   crash/src/app/view-accidents-container/view-map-accidents/search-map.component.ts
